### PR TITLE
Clarifications (batching, end-condition)

### PR DESCRIPTION
**Pull request by eladalon1983**
***
<a href="https://pr-preview.s3.amazonaws.com/screen-share/captured-mouse-events/pull/21.html" title="Last updated on Jul 3, 2023, 12:47 PM UTC (b7d8f06)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/screen-share/captured-mouse-events/21/29a8c12...b7d8f06.html" title="Last updated on Jul 3, 2023, 12:47 PM UTC (b7d8f06)">Diff</a>

### CODE REVIEW

**Comment by eladalon1983** [comment]
@fred-wang, could you please review?

File: index.html [hunk 1]
```diff
@@ -167,9 +167,14 @@ <h2>CaptureController Extensions</h2>
             We extend {{CaptureController}} with {{CaptureController/oncapturedmousechange}}, an
             {{EventHandler}} for events of type {{CapturedMouseEvent}}.
           </p>
-          <p>Events MUST NOT be fired more than 30 times per second.</p>
           <p>
-            Events MUST NOT be fired after the last clone of the original video track is stopped.
+            User agents MAY limit the frequency with which events are fired. This can be achieved by
+            briefly buffering events, then skipping the event for one state and reporting only the
```
Comment (fred-wang) [inline index.html]: Sorry, I don't really get this sentence especially the "skipping the event for one state"? I mean it looks similar to what I implemented in the prototype but I don't understand the wording.

**Review by fred-wang — COMMENTED** [review]
This LGTM but there some wording seems weird to me.

File: index.html (hunk 1 above)
Comment (eladalon1983) [reply index.html]: How about "then only dispatching the last event in the buffer"?

Comment (fred-wang) [reply index.html]: Yes, as a replacement to "then skipping the event for one state and reporting only the latest one".

**Review by fred-wang — APPROVED** [review]

**Pull request merged**